### ISSUE TITLE
Debian-Package .deb

### ISSUE THREAD

**Issue by MarcSiemering** [OpenRoberta/robertalab-usbprogram-nxt#5]
Could you please provide a debian-package .deb for the linux installation.
This would really help us for distributing the software to all PCs in our school system.
Thanks.

**Comment by catedrastics** [comment]
Yes, I support the request. A debian package installer and correct the i386 architecture that is no longer used in raspberry.
